Don't try to make the app bundle. It fails if the port isn't already installed,
and the resulting app bundle isn't installed anywhere.
https://github.com/Velocidex/c-aff4/issues/17
--- tools/pmem/Makefile.am.orig	2018-02-12 16:11:48.000000000 -0600
+++ tools/pmem/Makefile.am	2018-03-28 10:44:16.000000000 -0500
@@ -63,16 +63,4 @@
 osxpmem_SOURCES = pmem_imager.cc osxpmem.cc
 osxpmem_LDADD = $(top_srcdir)/aff4/libaff4.la -lyaml-cpp
 
-# In order to build a deployable bundle we must have all the libraries
-# installed on the system first.
-install-exec-hook:
-	@echo Creating deployable bundle.
-	rm -rf osxpmem.app osxpmem.zip
-	mkdir osxpmem.app/
-	cp README.md $(bindir)/osxpmem osxpmem.app/  || echo You must run make install before building a bundle.
-	dylibbundler -x osxpmem.app/osxpmem -b -d osxpmem.app/libs/ -p @executable_path/libs/ -of -od -cd
-	cp -a resources/MacPmem.kext osxpmem.app/
-	chmod -R o-wx osxpmem.app/
-	zip -r osxpmem.zip osxpmem.app/
-
 endif
